webpack.prod.js

```diff
@@ -0,0 +1,28 @@
+const webpack = require("webpack");
```

Comment:
Nice!  I haven't crossed the bridge of getting set up for production yet.  You're really getting a head start on that; very cool.